ibm-qradar-edr/info.json

```diff
@@ -0,0 +1,823 @@
+{
+  "name": "ibm-qradar-edr",
+  "label": "IBM QRadar EDR",
+  "version": "1.0.0",
+  "description": "IBM Security QRadar EDR, formerly ReaQta, remediates known and unknown endpoint threats in near real time with easy-to-use intelligent automation that requires little-to-no human interaction. ",
+  "publisher": "Fortinet",
+  "cs_approved": false,
+  "cs_compatible": true,
+  "category": "Endpoint Security",
+  "icon_small_name": "",
+  "icon_large_name": "",
+  "help_online": "",
+  "configuration": {
+    "fields": [
+      {
+        "name": "server_url",
+        "title": "Server URL",
+        "type": "text",
+        "editable": true,
+        "visible": true,
+        "required": true,
+        "description": "Specify the Hive server URL to which you will connect and perform the automated operations.",
+        "placeholder": "https://<Hive Server URL>"
+      },
+      {
+        "name": "secret_key",
+        "title": "Secret Key",
+        "type": "text",
+        "editable": true,
+        "visible": true,
+        "required": true,
+        "tooltip": "Specify the Secret Key of the API Application already created in the Hive Server. For more information refer https://www.ibm.com/support/pages/node/6568259",
+        "description": "Specify the Secret Key of the API Application already created in the Hive Server. For more information refer https://www.ibm.com/support/pages/node/6568259"
+      },
+      {
+        "name": "app_id",
+        "title": "App ID",
+        "type": "password",
+        "editable": true,
+        "visible": true,
+        "required": true,
+        "tooltip": "Specify the App ID of the API Application already created in the Hive Server. For more information refer https://www.ibm.com/support/pages/node/6568259",
+        "description": "Specify the App ID of the API Application already created in the Hive Server. For more information refer https://www.ibm.com/support/pages/node/6568259"
+      },
+      {
+        "name": "verify_ssl",
+        "title": "Verify SSL",
+        "type": "checkbox",
+        "editable": true,
+        "visible": true,
+        "required": false,
+        "value": true,
+        "description": "Specifies whether the SSL certificate for the server is to be verified or not. \nBy default, this option is set as True. "
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "get_alert_list",
+      "title": "Get Alert List",
+      "annotation": "get_alert_list",
+      "description": "This action is used to retrieve all alerts generated in ReaQta Hive.",
```

Comment:
ReaQta here is QRadar EDR i guess. Can we stick with any one of the name for uniformity.

Reply:
Changed